### DISCUSSION TITLE
第5节课作业(14班)

### DISCUSSION THREAD

**Discussion by NewDaNew** [InternLM/Tutorial#307] [作业]
冲！！

**Comment by ZZfive** [comment]
https://sincere-papyrus-971.notion.site/706142b9891f4887a8e521f5e6ef58de?pvs=4

**Comment by Nobitaxi** [comment]
https://blog.csdn.net/weixin_44288092/article/details/135696601

**Comment by Mercuryoo** [comment]
https://t3d0h0gdxm.feishu.cn/docx/Q10Pd2xG1obIJOx6kHYc6VswnZe?from=from_copylink

**Comment by xilimala** [comment]
https://zhuanlan.zhihu.com/p/679223733

**Comment by shiningwhite-cmd** [comment]
https://www.yuque.com/shanxiaobai/xt4xo5/ivye3db24zqxdqa6?singleDoc# 《LMDeploy的使用》

**Comment by SingaCycle** [comment]
https://blog.csdn.net/z1370378301/article/details/135886733?spm=1001.2014.3001.5501 LMdeploy笔记+作业

**Comment by lizihe11** [comment]
https://gitee.com/kiddylzh/intern-lm-tutorial-note/blob/master/task/fifth%20task.md
